monitor.js

```diff
@@ -0,0 +1,117 @@
+const fetch = require('node-fetch')
+const config = require('./monitor.json');
+var env=null;
+var monitorEnable=null;
+var axiomApiToken=null;
+var axiomIngestApiUrl=null;
+var platform=null;
+
+export default function monitor(app) {
+    const monitorConfig=config[app.get("env")].monitor;
+    env=monitorConfig.env;
+    monitorEnable=monitorConfig.enable;
+    axiomApiToken=monitorConfig.axiomApiToken;
+    axiomIngestApiUrl=monitorConfig.axiomIngestApiUrl;
+    platform=monitorConfig.platform;
+
+    app.post('/installed', async (req, res, next) => {
+        await ingest_installed(req.body);
```

Comment:
Remove `await` as it is not necessary to wait

Reply:
I use [Middleware Extension Routing] (https://expressjs.com/en/guide/usion-middle.html#middle.router), async+await is to not block the original/Installed request?

Reply:
I don't know much about express middlewares, but if it dosen't block any subsequent middleware, then what is the `next` running for? And if you are not waitting for anything then the await are unnecessary, regardless of whether it blocks things or not.
```
ingest_installed(req.body);
next();
```